samples: ipc_service: Restore default configuration (#16149)
nrfconnect/sdk-nrf: samples/ipc/ipc_service/README.rst

**nRF5340 DK**

You can build the sample using either the RPMsg or ICMSG backends, as follows:

> The RPMsg backend is the default one, you can use the following command:

Reviewer: Just suggesting a bit different wording. Still @peknis has a final word on this.
```suggestion
For the RPMsg backend, that is the default one, you can use the following command:
```

Reviewer 2: Now, you have two sentences with colon at the end, which is not good. Suggesting the following:

You can build the sample using either the RPMsg or ICMSG backend.
For the default RPMsg backend, use the following command:

.. code-block:: console

   west build -p -b nrf5340dk/nrf5340/cpuapp

> For the ICMSG backend, you can use the following command:

Reviewer 2: ```suggestion
For the ICMSG backend, use the following command:
```

**nRF54H20 DK**

> You can build the sample to test IPC between the application and radio domains using the :ref:`zephyr:ipc_service_backend_icbmsg` backend that is the default sample configuration, as follows:

Reviewer 2: ```suggestion
To build the sample to test IPC between the application and radio domains using the default :ref:`zephyr:ipc_service_backend_icbmsg` backend, use the following command:
```

.. code-block:: console

   west build -p -b nrf54h20dk/nrf54h20/cpuapp

> You can build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, as follows:

Reviewer 2: ```suggestion
To build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, use the following command:
```
